script.js

```diff
@@ -1,4 +1,73 @@
+//global variables
+var roll2Dice = 'Player starts to roll 2 dice';
+var concatenateRandomDiceDigits = 'Player chooses Dice 1 or Dice 2 to be ordered first';
+var gameMode = roll2Dice;
+var userValue;
+var dice1;
+var dice2;
+var playerNumber = 1;
+var player1Value;
+var player2Value;
+
+// diceroll function 
+var rollDice = function () {
+  var randomInteger = Math.random() * 6;
+  var randomRoll = Math.floor(randomInteger) + 1;
+  return randomRoll;
+};
+
+// roll 2 dice function
+var roll2DiceFunction = function(){
+    dice1 = rollDice();
+    dice2 = rollDice();
+    console.log('Dice1:' + dice1);
+    console.log('Dice2:' + dice2);
+    var myOutputValue =`Player ${playerNumber} rolled 2 dice! Dice 1 rolled a ${dice1} <br>Dice 2 rolled a ${dice2}. <br>Now, choose <b>Dice 1</b> or <b>Dice 2</b> for the first digit.`;
+    return myOutputValue;
+  };
+
+// choose which dice value goes first
+var createValue = function (input){
+  
+   if (input== 'Dice 1'){
+     userValue = String(dice1)+String(dice2);
+   } else if (input == 'Dice 2'){
+     userValue = String(dice2)+String(dice1);
+   };
+   console.log ('uservalue:'+userValue);
+   var myOutputValue = `Player ${playerNumber} chose ${input} to go first! Your value is ${userValue}!`;
+   return myOutputValue;
+};
+
 var main = function (input) {
-  var myOutputValue = 'hello world';
-  return myOutputValue;
+  if (gameMode == roll2Dice && playerNumber == 1){
```

Comment:
I notice that you are using && statements. I would suggest using nested if statements.